robot/Cumulus/resources/AdvancedMappingPageObject.py

```diff
@@ -0,0 +1,78 @@
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+from NPSP import npsp_lex_locators
+from logging import exception
+
+@pageobject("Custom", "ManageAdvancedMapping")
+class AdvancedMappingPage(BasePage):
+
+    
+    @property
+    def npsp(self):
+        return self.builtin.get_library_instance('NPSP')
+    
+    def create_new_field_mapping(self,src_fld,tgt_fld):
+        self.selenium.wait_until_page_contains("Field Mappings", timeout=60)
+        self.selenium.scroll_element_into_view(npsp_lex_locators['adv_mappings']['button'])
+        btns=npsp_lex_locators['adv_mappings']['button']
+        self.selenium.click_button(btns)
+        self.selenium.wait_until_page_contains_element(npsp_lex_locators['adv_mappings']["modal_open"], timeout=15, error="New Field Mapping Modal did not open in 15 seconds")
+        src_loc=npsp_lex_locators['adv_mappings']['field_mapping'].format("sourceFieldLabel")
+        self.selenium.click_element(src_loc)
+        self.selenium.wait_until_page_contains_element(npsp_lex_locators['adv_mappings']['combobox'], error="Source field dropdown did not open")
+        src_ele=npsp_lex_locators['span'].format(src_fld)
+        element = self.selenium.driver.find_element_by_xpath(src_ele)
+        self.selenium.driver.execute_script('arguments[0].click()', element) 
+        target=npsp_lex_locators['adv_mappings']['field_mapping'].format("targetFieldLabel")
+        self.selenium.click_element(target)
+        self.selenium.wait_until_page_contains_element(npsp_lex_locators['adv_mappings']['combobox'], error="Target field dropdown did not open")
+        self.selenium.click_element(npsp_lex_locators['span'].format(tgt_fld))
```

Comment:
Just curious. You are defining variable for some of the locators and for some, using it directly. Any specific reason ?